rootfs-builder/rootfs.sh

```diff
@@ -550,6 +629,26 @@ parse_arguments()
 
 	shift $(($OPTIND - 1))
 	distro="$1"
+	arch=$(uname -m)
+
+	if [ "${distro}" == "alpine" -o "${distro}" == "euleros" ]; then
+		if [ "${RUST_AGENT}" == "yes" ]; then
+			die "rust agent cannot be built on ${distro}.
```

Comment:
Again, it's great that you've provided this detail but I think it would be better to:

- Create a GitHub issue with the full details.
- Reference that issue URL here with a brief description.

Something like:

> Building the rust agent on Alpine is not supported - see http://...